resources/webview-ui/styles.css

```diff
@@ -2,28 +2,30 @@ h1 {
   font-size: 1.5em;
 }
 
-#search-container {
+.search-container {
   display: flex;
   flex-direction: column;
   justify-content: center;
 }
 
-#results-container {
+/* There's no delement with id "results-container", what was it for */
```

Comment:
@vudayani-vmw

Reply:
@Eskibear 
 Thanks for pointing it out. These stylings are redundant and should be removed.